CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(backtrace)
+
+# Automake uses -frandom-seed initialized with file name of given file
+# but AFAIK it can't be done on CMake, so here's always same seed
+set(CMAKE_CXX_FLAGS "-DHAVE_CONFIG_H -funwind-tables -frandom-seed=mySeed -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition -Wmissing-format-attribute -Wcast-qual -g -O2")
+set(CMAKE_C_FLAGS "-DHAVE_CONFIG_H -funwind-tables -frandom-seed=mySeed -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition -Wmissing-format-attribute -Wcast-qual -g -O2")
+
+file(GLOB sources 
+	atomic.c dwarf.c fileline.c posix.c print.c sort.c state.c backtrace.c
+	simple.c pecoff.c read.c alloc.c config.h
+)
+file(GLOB export_headers
+	${CMAKE_CURRENT_BINARY_DIR}/backtrace.h
+	${CMAKE_CURRENT_BINARY_DIR}/backtrace-supported.h
+)
+
+add_library(${PROJECT_NAME} ${sources} ${export_headers})
+include_directories(${PROJECT_NAME} ${CMAKE_CURRENT_BINARY_DIR})

```

Comment:
Instead, prefer setting the variable `set(CMAKE_INCLUDE_CURRENT_DIR ON)` at the top of the project